README.md

```diff
@@ -1 +1,61 @@
-# school_library
\ No newline at end of file
+# SCHOOL LIBRARY
+
+> Imagine that you are the librarian of OOP University, and you need a tool to record what books are in the library and who borrows them. The app that you will create will allow a you to:  
+- Add new students or teachers. 
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+## Project screenshots
+
+![screenshot(COMING SOON)](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- RUBY
+
+## How to Setup
+
+> You can simply clone or download [this repository](https://github.com/Mosams/school_library.git), and use your favorite browser or code editor to run this program.
+
+- To open the project after download, simply double click the index.html file
+
+- To open this project using vs code ( for this example) or your favorite code editor, you can follow the guide below:
+  > in your cmd or command line navigate to where this project is located, then;
+
+```cmd
+cd school_library
+```
+
+> thereafter run
+
+```cmd
+code .
+```
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.